features/step_definitions/templates.js

```diff
@@ -5,7 +5,7 @@ const request = require('../support/request');
 const path = require('path');
 const Assert = require('chai').assert;
 
-module.exports = function () {
+module.exports = function server() {
```

Comment:
This can just be `module.exports = () => {`

Same as other files. I'm not sure why we had it as `function server() {` in the first place